docs/content/concepts/io-management/io-managers.mdx

```diff
@@ -821,7 +821,7 @@ Additionally, if the handled output is part of a software-defined asset, these m
 | <PyObject module="dagster" object="InMemoryIOManager" />                                   | Stores outputs in memory. Primarily useful for unit testing.                  |                                                                                                                        |
 | <PyObject module="dagster_aws.s3" object="S3PickleIOManager" />                            | Stores outputs as pickle files in Amazon Web Services S3.                     |                                                                                                                        |
 | <PyObject module="dagster_azure.adls2" object="ConfigurablePickledObjectADLS2IOManager" /> | Stores outputs as pickle files in Azure ADLS2.                                |                                                                                                                        |
-| <PyObject module="dagster_gcp.gcs" object="ConfigurablePickledObjectGCSIOManager" />       | Stores outputs as pickle files in Google Cloud Platform GCS.                  |                                                                                                                        |
+| <PyObject module="dagster_gcp.gcs" object="GCSPickleIOManager" />                          | Stores outputs as pickle files in Google Cloud Platform GCS.                  |                                                                                                                        |
```

Comment:
Same here as on the S3 PR - the PyObject link isn't working for me. Potential Sphinx thing?

Reply:
All good here!